src/app/components/signin/signin.component.ts

```diff
@@ -66,7 +66,7 @@ export class SigninComponent implements OnInit {
               message: 'Logged in successfully! Redirecting...'
             };
 
-            this.router.navigateByUrl('/channels');
+            this.router.navigateByUrl('/campaigns');
```

Comment:
This is unfortunately not correct. Default page should still be channels. David only wanted the Campaigns to be renamed to "Crowdfunding" in the header.